server/api/forms/index.js

```diff
@@ -0,0 +1,31 @@
+import Router from 'koa-router'
+const pg_db = require('./db')
+
+const router = Router()
+
+router
+  .get('/specific', async(ctx, next)=>{
+    var test = {};
+    // Trying to force the function to wait on the result of the query
+    await pg_db.query("SELECT * FROM \"Form_Info\" WHERE formname = 'test'", (err, res2)=>{
```

Comment:
Not sure whether this is the correct way of using `async/await`. I'll play with the code on my machine and get back to you about it.